.editorconfig

```diff
@@ -4,7 +4,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
why make this change? All the rest of the UI is 2 spaces.

Reply:
ohh my mistake. let me revert it.